webapp/Connector/src/view/GroupSummary.js

```diff
@@ -210,8 +210,22 @@ Ext.define('Connector.view.GroupSummaryBody', {
                     html: '<div class="module"><h3>In the plot</h3></div>'
                 });
                 rightColumn.add(Connector.view.PlotPane.plotFilterContent(plotFilter));
+                rightColumn.add({
+                    xtype: 'button',
+                    text: 'View in Plot',
+                    itemId: 'groupplotview',
+                    style: 'margin-top: 20px'
+
+                })
             }
         }
+        else
```

Comment:
I would move var plotFilter to outside of if condition, so that a group will either show 'View in Plot' or 'No plot saved...'. Though unlikely to happen in real life, the current if/else grouping can end up showing neither if 'containsPlot' is true but plotFilter is null.